pages/docs/modal-system.en-US.mdx

```diff
@@ -0,0 +1,71 @@
+import { Callout } from "nextra-theme-docs";
+
+# The modal system
+
+An easier way to trigger a modal without having to have a conditional state to render it in the DOM.
```

Comment:
```suggestion
The modal system is a part of the framework that handles rendering modal dialogs, warnings, and other similar displays. Its main goal is to coordinate modals across different applications so that only a single modal is rendered for the user at a time without any other modals being removed.
```